src/main/java/com/exasol/adapter/jdbc/BaseTableMetadataReader.java

```diff
@@ -141,32 +139,25 @@ protected void logSkippingUnsupportedTable(final String tableName) {
         LOGGER.fine(() -> "Skipping unsupported table \"" + tableName + "\" when mapping remote metadata.");
     }
 
-    private boolean isTableSelected(final List<String> selectedTables, final String tableName) {
-        if (checkIfTableIsSelected(tableName, selectedTables)) {
-            return isTableIncludedByFilter(tableName);
+    private boolean isFilteredTable(final List<String> filteredTables, final String tableName) {
+        if (isFilteredByProperties(tableName) && isFiltered(tableName, filteredTables)) {
```

Comment:
If we group the checks like that we lose an opportunity to log if the table was filtered by request parameters or user filter. Do you think i's fine to have only one log message?
If we keep only one message, I suggest to improve the message and write that "a table was filtered out by request properties OR a user filter" or something like this